marimo/_pyodide/pyodide_session.py

```
@@ -318,6 +340,7 @@ def _launch_pyodide_kernel(
     control_queue: asyncio.Queue[ControlRequest],
     set_ui_element_queue: asyncio.Queue[SetUIElementValueRequest],
     completion_queue: asyncio.Queue[CodeCompletionRequest],
+    mcp_evaluation_queue: asyncio.Queue[MCPServerEvaluationRequest],
```

Ruff (0.8.2)

Unused function argument: `mcp_evaluation_queue`

(ARG001)